src/software/ai/hl/stp/skill/dribble/dribble_skill_fsm.h

```diff
@@ -105,112 +91,121 @@ struct DribbleFSM
         std::optional<Angle> final_dribble_orientation_opt);
 
     /**
-     * Action to get possession of the ball
+     * Action to get control of the ball
      *
      * If the ball is moving quickly, then move in front of the ball
      * If the ball is moving slowly, then chase the ball
      *
-     * @param event DribbleFSM::Update
+     * @param event the Update event
      */
-    void getPossession(const Update &event);
+    void getBallControl(const Update &event);
 
     /**
      * Action to dribble the ball
      *
      * This action will orient the robot towards the destination, dribble to the
      * destination, and then pivot to face the expected orientation
      *
-     * @param event DribbleFSM::Update
+     * @param event the Update event
      */
     void dribble(const Update &event);
 
     /**
-     * Start dribbling
-     *
-     * @param event DribbleFSM::Update
-     */
-    void startDribble(const Update &event);
-
-    /**
-     * Action to lose possession of the ball
+     * Action to lose BallControl of the ball
      *
-     * @param event DribbleFSM::Update
+     * @param event the Update event
      */
     void loseBall(const Update &event);
 
     /**
-     * Guard that checks if the robot has possession of the ball
+     * Guard that checks if the robot has control of the ball
      *
-     * @param event DribbleFSM::Update
+     * @param event the Update event
      *
-     * @return if the ball has been have_possession
+     * @return if the robot has control of the ball
      */
-    bool havePossession(const Update &event);
+    bool haveBallControl(const Update &event);
 
     /**
-     * Guard that checks if the robot has lost possession of the ball
+     * Guard that checks if the robot has lost control of the ball
      *
-     * @param event DribbleFSM::Update
+     * @param event the Update event
      *
-     * @return if the ball possession has been lost
+     * @return if the robot has lost control of the ball
      */
-    bool lostPossession(const Update &event);
+    bool lostBallControl(const Update &event);
```

Comment:
why do we need two separate guards? Can they not be inverses of each other?

Reply:
`lostBallControl` uses `lose_ball_control_threshold` which is a little more lenient